dropwizard-configuration/src/test/java/io/dropwizard/configuration/EnvironmentVariableLookupTest.java

```diff
@@ -1,14 +1,14 @@
 package io.dropwizard.configuration;
 
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.DisabledIf;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.assertj.core.api.Assumptions.assumeThat;
 
 class EnvironmentVariableLookupTest {
     @Test
+    @DisabledIf("isNopeEnvVarSet")
```

Comment:
Could we use [`@DisabledIfEnvironmentVariable`](https://junit.org/junit5/docs/current/api/org.junit.jupiter.api/org/junit/jupiter/api/condition/DisabledIfEnvironmentVariable.html) for this? If the value is `null`, the annotation should have no effect, so this should be what we want.

Reply:
I think we can actually deprecate and remove `EnvironmentVariableLookup`... PR coming right up..

Reply:
See what you think of https://github.com/dropwizard/dropwizard/pull/5188